[build] Fix docker-sonic-vs build issue for missing pygobject.

liushilongbuaa opened the pull request:
<!--
     Please make sure you've read and understood our contributing guidelines:
     https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

     ** Make sure all your commits include a signature generated with `git commit -s` **

     If this is a bug fix, make sure your description includes "fixes #xxxx", or
     "closes #xxxx" or "resolves #xxxx"

     Please provide the following information:
-->

#### Why I did it
1. We already use apt to install python3-gi in SONiC image according to #22326 
But docker-sonic-vs.gz change is missed in that PR.

##### Work item tracking
- Microsoft ADO **(number only)**:  32869072

#### How I did it
1. Install python3-gi in docker-sonic-vs.gz image.
#### How to verify it

<!--
If PR needs to be backported, then the PR must be tested against the base branch and the earliest backport release branch and provide tested image version on these two branches. For example, if the PR is requested for master, 202211 and 202012, then the requester needs to provide test results on master and 202012.
-->

#### Which release branch to backport (provide reason below if selected)

<!--
- Note we only backport fixes to a release branch, *not* features!
- Please also provide a reason for the backporting below.
- e.g.
- [x] 202006
-->

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

<!--
- Please provide tested image version
- e.g.
- [x] 20201231.100
-->

- [ ] <!-- image version 1 -->
- [ ] <!-- image version 2 -->

#### Description for the changelog
<!--
Write a short (one line) summary that describes the changes in this
pull request for inclusion in the changelog:
-->

<!--
 Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.
-->

#### Link to config_db schema for YANG module changes
<!--
Provide a link to config_db schema for the table for which YANG model
is defined
Link should point to correct section on https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/doc/Configuration.md
-->

#### A picture of a cute animal (not mandatory but encouraged)

wangxin commented:
Why disable unstable nvidia platform in version upgrade pipeline? Is there a plan to fix the instability?

mssonicbld commented:
/azp run Azure.sonic-buildimage

mssonicbld commented:
/azp run Azure.sonic-buildimage

liushilongbuaa commented:
> nvidia platform in version upgrade pipeline? Is there a plan to fix

I checked the fix process. nvidia fixed it 13 hours ago.
pensando fix is still in progress but it seems only needs a few days to finish.
Removed pipeline change in this PR.

mssonicbld commented:
/azp run Azure.sonic-buildimage

wangxin commented:
> > nvidia platform in version upgrade pipeline? Is there a plan to fix
> 
> I checked the fix process. nvidia fixed it 13 hours ago. pensando fix is still in progress but it seems only needs a few days to finish. Removed pipeline change in this PR.

Because the pensando fix is in progress, why need to skip pensando in this PR? If we don't skip it here, are we blocked?

mssonicbld commented:
/azp run Azure.sonic-buildimage

liushilongbuaa commented:
> > > nvidia platform in version upgrade pipeline? Is there a plan to fix
> > 
> > 
> > I checked the fix process. nvidia fixed it 13 hours ago. pensando fix is still in progress but it seems only needs a few days to finish. Removed pipeline change in this PR.
> 
> Because the pensando fix is in progress, why need to skip pensando in this PR? If we don't skip it here, are we blocked?

I will confirm if pensando still needs public official build. Removed this change in this PR.

liushilongbuaa commented:
/azpw ms_conflict
